Workbooks/Azure Monitor - Workspaces/Prometheus Explorer/Prometheus Explorer.workbook

```diff
@@ -178,7 +178,7 @@
     {
       "type": 1,
       "content": {
-        "json": "You do not have query access to the selected Azure Monitor Workspace.  Please [visit the workspace permissions documentation](https://aka.ms/azuremonitorworkspacepermissions) for more information.",
+        "json": "You do not have query access to the selected Azure Monitor workspace. [Please visit the troubleshooting documentation for more information.](https://aka.ms/azuremonitorworkspacepermissions)",

```

Comment:
Sorry I think the first part ("You do not have query access to the selected Azure Monitor workspace.") may have already been agreed upon with Auston, but thinking again about this, depending upon what we hear from case reviews maybe this will want to update to something like "Your Azure Monitor workspace could not be reached" since network access is the most typical case recently, even though something like that is generic enough to still cover RBAC issues.

Reply:
(feel free to go ahead for now w/ current text though, this is definitely an improvement)